src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -1051,6 +1051,7 @@ codet java_bytecode_convert_methodt::convert_instructions(
        i_it->statement=="checkcast" ||
        i_it->statement=="newarray" ||
        i_it->statement=="anewarray" ||
+       i_it->statement==patternt("?div") ||
```

Comment:
only `idiv` and `ldiv` throw `ArithmeticException`, `fdiv` and `ddiv` do not. In addition `irem` and `lrem` throw, too.